### DISCUSSION TITLE
Connection between boards using 40 pin IDE cables

### DISCUSSION THREAD

**Discussion by ducky92** [sanni/cartreader#449] [Ideas & Suggestions]
Has there been any thought in using 40 pin ide cables as the medium to connect the 2 board layers instead of long GPIO Pins?
Something like these https://www.aliexpress.com/item/2255801037605883.html
I was playing around with case designs and thought it _may_ be easier to work with. I do not know anything about PCB designs so there could be a blatantly obvious reason as to why not to do this.

**Comment by Ancyker** [comment]
I see no reason something like that shouldn't work (though that specific one doesn't have the greatest reviews). Sanni said in another thread that mass production was the end goal, if so then simplifying the design to only need 1 or 2 PCBs would be better than using the cable. Having the slots and Arduino circuitry embedded on the main PCB would make it so that more could be done by machine. The display likely will always need to be its own thing but the rest can probably be integrated onto a single board. There may be a way to support both a mounted Arduino and the Arduino embedded SMD components at the same time, similar to how 2 types of chips are supported on the CIC board.

Edit: With everything on one board the 3.3V/5V switch might be able to be eliminated?
